spec/graphql/execution_error_spec.rb

```diff
@@ -320,6 +308,22 @@
       }
       assert_equal(expected_result, result)
     end
-  end
 
+    describe "more than one ExecutionError on a field defined to return a list" do
```

Comment:
One more test, similar to the earlier one, but this time the return type is a list type and so the path includes the list indices as well.